node-cheatsheet.md

```diff
@@ -12,5 +12,5 @@ A glossary of terms for Node.js.
 | `npm package`  | Open source Node project, published on npm. |
 | `npm init`     | Command run in the terminal to initialise your Node project and create a package.json file. |
 | `npm install <package-name>`  | Command run in the terminal to install an npm package. |
-| `package.json` | File holding various metadata about a Node project, such as project name, description or license information, installed packages (or dependencies). Usually located in the root directory. Package.json cheatsheet [here](http://browsenpm.org/package-json). |
+| `package.json` | File holding various metadata about a Node project, such as project name, description or license information, installed packages (aka dependencies). Usually located in the root directory. Package.json cheatsheet [here](http://browsenpm.org/package-json). |
 | `dependencies` | Packages installed from npm that are necessary to run the project successfully. Listed in the package.json. |
```

Comment:
Worth adding dev dependencies here too? They'll appear in most projects they look to for examples!